scripts/lintfix/index.js

```diff
@@ -0,0 +1,9 @@
+const argv = require('yargs').argv;
+const sh = require('shelljs');
+const tslintConfig = 'tslint.json';
+const tsConfig = argv.tsconfig || 'tsconfig.json';
+
+sh.exec(`node node_modules/tslint/bin/tslint --project ${tsConfig} --config ${tslintConfig} --fix`, () => {
```

Comment:
you can probably use `tslint --project ${tsConfig} --config ${tslintConfig} --fix` instead (without the node node_modules/tslint/bin/)